src/test/java/taskFinal/page/AddressBookPage.java

```diff
@@ -0,0 +1,70 @@
+package taskFinal.page;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.Select;
+import taskFinal.util.Address;
+
+public class AddressBookPage {
+
+    private WebDriver driver;
+
+    @FindBy(css="button[title='Add New Address']")
+    WebElement addNewAddressButton;
+
+    @FindBy(css = "input#telephone")
+    WebElement phoneNumberInput;
+
+    @FindBy(css = "input#street_1")
+    WebElement streetInput;
+
+    @FindBy(css = "input#city")
+    WebElement cityInput;
+
+    @FindBy(css = "select#region_id")
+    WebElement stateSelect;
+
+    @FindBy(css = "input#zip")
+    WebElement zipInput;
+
+    @FindBy(css = "button[title='Save Address']")
+    WebElement saveButton;
+
+    @FindBy(xpath = "//tbody/tr[1]")
+    WebElement addressTable;
+
+    public AddressBookPage(WebDriver driver) {
+        this.driver = driver;
+        PageFactory.initElements(driver, this);
+    }
+
+    public Address addNewAddress(Address address)
+    {
+        driver.manage().window().fullscreen();
```

Comment:
should be moved to driver configuration

Reply:
I set driver configuration in the SingleWebDriver class - driver.manage().window().fullscreen();
The problem when I get the address book page, the page became reduced.